packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -82,6 +97,30 @@
       </v-card-actions>
     </v-card>
   </v-dialog>
+
+  <v-dialog width="70%" v-model="contractStateDialog">
+    <v-card>
+      <v-card-title class="text-h5">Contract lock Detalis</v-card-title>
+      <v-card-text>
+        <p v-if="loading" class="text-center">
+          <strong>Loading The Locked Amount...</strong>
+        </p>
+        <p v-else class="text-center">
+          Amount Locked <strong>{{ contractLocked?.amountLocked }}</strong>
+        </p>
+        <br />
+        <v-alert type="info" variant="tonal">
+          To unlock your contract from the grace period, please ensure that you fund your account with the displayed
```

Comment:
The contract is in a `GracePeriod` condition, which means that your workloads are suspended but not deleted; in order to resume your workloads and restore their functionality, you must pay your account with the necessary tokens.